pkgs/unified_analytics/test/unified_analytics_test.dart

```diff
@@ -177,11 +178,18 @@ void main() {
     ) as FakeAnalytics;
     analytics.clientShowedMessage();
 
+    // Invoking a send command should reset the session file to a good state
+    //
+    // Having it reformat the session file before any send event happens will just
+    // add additional work on startup
+    analytics.send(testEvent);
+
```

Comment:
This test shows that we are now allowing the session file to be in a bad state until we invoke a send command.